app/components/SystemData.jsx

```diff
@@ -16,6 +17,52 @@ class SystemData extends Component {
         ? 'None'
         : (sum / numOfDataPoints).toFixed(2);
 
+    const chartData = {
```

Comment:
is this whats considered "in-line" styling? If so, can it be all done on  .scss file to keep this cleaner? It seems that its a mix...as if this is how chart.js works, but I cant be certain.

Reply:
Good question! It's not in-line styling, but it's very similar. In-line styling would be something like this: <div style="background-color: blue">